tests/config/pytest_software.yml

```diff
@@ -0,0 +1,748 @@
+abacas:
+  - software/abacas/**
```

Comment:
This file needs to be removed from the PR.

Reply:
Removed file.